src/scripts/tweaks/caught_up_line.js

```diff
@@ -0,0 +1,41 @@
+import { keyToCss } from '../../util/css_map.js';
+import { buildStyle } from '../../util/interface.js';
+import { onNewPosts } from '../../util/mutations.js';
+
+const hiddenClass = 'xkit-tweaks-caught-up-line-title';
+const borderClass = 'xkit-tweaks-caught-up-line-border';
+
+const styleElement = buildStyle(`
+  .${hiddenClass} > div { display: none; }
+  .${borderClass} > div > div { border-top: 4px solid rgb(var(--white-on-dark)); height: 0px; }
+`);
+
+let tagChicletCarouselItemSelector;
+let listTimelineObjectSelector;
+
+const createCaughtUpLine = async function () {
+  [...document.querySelectorAll(tagChicletCarouselItemSelector)]
+    .map(tagChicletCarouselItem => tagChicletCarouselItem.closest(listTimelineObjectSelector))
```

Comment:
```suggestion
    .map(tagChicletCarouselItem => tagChicletCarouselItem.closest(listTimelineObjectSelector))
    .filter((element, index, array) => array.indexOf(element) === index)
```

Does this work? This seems to be the easiest way to remove duplicate items from an array without interrupting the chain, but I'm not at liberty to test its implementation today.

Reply:
I don't think it'll make a performance difference (anything run once in onNewPosts will be 0ms) but it's a fun little Javascript problem to solve!

`new Set(items.map(/* */).forEach(/* */)` is probably the clearest way semantically, given that the reader knows what an ES6 Set does.